doc/source/cluster/kubernetes/examples/ml-example.md

```diff
@@ -11,9 +11,16 @@ at the {ref}`introductory guide <kuberay-quickstart>` first.
 In this guide, we show you how to run a sample Ray machine learning
 workload on Kubernetes infrastructure.
 
-We will run Ray's {ref}`XGBoost training benchmark <xgboost-benchmark>` with a 100 gigabyte training set.
+We will run Ray's {ref}`XGBoost training benchmark<xgboost-benchmark>` with a 100 gigabyte training set.
```

Comment:
Eh, is this change intentional?
In general I'm slightly fuzzy on our final plan to finalize the docs.
cc @scv119